.github/workflows/ci-integration.yml

```diff
@@ -0,0 +1,54 @@
+name: Test ASReview webapp (integration)
+on: [push, pull_request]
```

Comment:
Since these tests are not *super* lightweight, I could imagine doing some things to optimize the workflow run:

* run it after the normal tests for `ci-webapp`. That way, if the normal tests fail, integration tests are not run. Plus compiling assets will already have been done.
* if we keep this in a separate workflow, run `ruff` in front of the tests, so they don't run on syntax errors etc.?
* *or* run the tests for PRs only if certain files have been altered, e.g. the JS. Not sure about this, since API changes can also break integration tests.